examples/logging.properties

```diff
@@ -1,3 +1,5 @@
+# Root log level (set to the lowest level needed by one or more loggers)
+.level=FINE
```

Comment:
Please make this build.buildfarm.level - I'd rather not direct folks to enable logging outside of our namespace